src/main/java/Clank.java

```diff
@@ -0,0 +1,69 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Clank {
+    private static TaskManager taskManager = new TaskManager();
+
+    public static void main(String[] args) {
+        String line = "____________________________________________________________\n";
+        String welcomeMessage = " Greetings! I'm Clank, Ratchet's best friend.\n" +
+                " I heard you are in trouble, and I'm here to help.\n" +
+                " How may I assist you?\n" +
+                line;
+
+        String byeMessage = " Bye. Hope to see you again soon!\n";
+        System.out.println(welcomeMessage);
+
+        Scanner scanner = new Scanner(System.in);
```

Comment:
The `Scanner `object is never closed, leading to a potential resource leak.

Use a `try-with-resources` statement to ensure `scanner` is automatically closed.

```suggestion
        try (Scanner scanner = new Scanner(System.in)) {
    while (true) {
        String input = scanner.nextLine();
        // (rest of the loop remains unchanged)
    }
}
```

Reply:
Fixed!